modules/manage/pages/manage-security/configure-ldap.adoc

```diff
@@ -188,96 +212,119 @@ For example, a rule-set might be expressed as follows:
   }
 ]
 ----
-+
+
 Given this rule-set, if the username `myuser@test.com` is specified, a match is made on the regular expression provided by the _first_ rule in the array: therefore the query `ou=users, dc=test,dc=com??one?(uid=myuser)` is used to retrieve the DN.
-+
+
 If the username `myuser` is specified, a match is made on the regular expression provided by the _second rule_: therefore, the template `cn&#61;&#x007B;0&#x007D;,ou&#61;users,dc&#61;example,dc&#61;com` is used to retrieve the DN.
-+
+
+[#testing-user-authentication]
+==== Testing User Authentication
+
 *Test User Authentication*, when opened, provides options for testing the authentication of specific users:
-+
+
 [#test-user-auth-field]
 image::manage-security/testUserAuth.png[,400,align=left]
-+
+
 Enter the username and password for the user, and left-click on *Test User Authentication*.
 Couchbase Server maps the specified username to an LDAP DN, and performs authentication on the LDAP server.
 Notifications confirming success or failure duly appear on the dialog.
 
 [#enable-ldap-group-authorization]
-* *Enable LDAP group authorization & sync*.
+[#group-authorization-enablement]
+=== Group Authorization Enablement
+
+Fields for enabling _LDAP Group Support_ are made visible by means of the *Enable LDAP group authorization & sync* switch.
+Enablement means that Couchbase Server retrieves from the *LDAP Hosts(s)* all _LDAP group membership_ information for users that the *LDAP Host(s)* authenticate.
```

Comment:
Couchbase Server doesn't retrieve all LDAP group membership info. We retrieve group membership info on per user basis only when authorization for this particular is being done (I think it can be called "on-demand").

Reply:
I have changed it to:

Enablement means that when a user has been authenticated by the *LDAP Hosts(s)*, Couchbase Server retrieves the _LDAP group membership_ information for that user, in order to authorize them.